src/java/picard/sam/DownsampleSam.java

```diff
@@ -24,10 +28,15 @@
  * of both ends of a pair or neither end of the pair!
```

Comment:
expand javadoc to state that also extra alignments (not only paired ends) will be included or excluded